tests/docker_scripts/stateless_runner.sh

```diff
@@ -243,6 +243,10 @@ function prepare_stateful_data() {
             ENGINE = CollapsingMergeTree(Sign) PARTITION BY toYYYYMM(StartDate) ORDER BY (CounterID, StartDate, intHash32(UserID), VisitID)
             SAMPLE BY intHash32(UserID) SETTINGS index_granularity = 8192, storage_policy='s3_cache'"
 
+        # Avoid locking filesystem cache
+        clickhouse-client --query "SYSTEM STOP MERGES test.hits"
```

Comment:
I'd rather not stop merges, but use a its own cache for tests which fail if it actually does not matter for the test (there are tests which on purpose use a common cache to generate more load/concurrency with it). In this exact case for tests which were modified in this PR it does not matter.

Reply:
> 02286_drop_filesystem_cache assumes that there will be no cache entries after SYSTEM DROP FILESYSTEM CACHE, but, due to very slow merges for test.hits some entries has not been removed since they are still in use.

yes, but there is no need to use the same cache object in test `02286_drop_filesystem_cache` as for `test.hits`, so with independent caches we do not care about slow merges of `test.hits`. `system.filesystem_cache` has a `name` columns, which is a name of the cache, so we can check only a specific cache in `02286_drop_filesystem_cache`, so `test.hits` will not have any impact on it.